### PR TITLE
Show debug information when LES is selected (in the titlebar).

### PR DESCRIPTION
**Pull request by mandrigin**
To simplify testing and debugging, there is a very simple sync indicator
(debug) for LES. This UI is not even close to being production-ready and will
be reworked in the future.

status: ready

### CODE REVIEW

**Comment by ghost** [comment]
### Pull Request Checklist
- [x] Have you updated the documentation, if impacted (e.g. [docs.status.im](https://docs.status.im/docs/build_status.html))?

**Review by janherich — APPROVED** [review]

**Comment by mandrigin** [comment]
@yenda @rasom this one is fixed, rebased, solved conflicts and it is pretty much ready to be reviewed/tested.

File: src/status_im/events.cljs [hunk 1]
```diff
@@ -113,6 +115,14 @@
  (fn [cofx _]
    (init/initialize-keychain cofx)))
 
+;; home screen
+
+(handlers/register-handler-fx
+ :home.ui/sync-info-pressed
+ (fn [cofx name]
+   (log/debug "igorm -> sync info pressed" name)
```
Comment (yenda) [inline src/status_im/events.cljs]: remove debug statement ?

File: src/status_im/ui/screens/home/subs.cljs [hunk 2]
```diff
@@ -6,3 +6,15 @@
                   :<- [:browser/browsers]
                   (fn [[chats browsers]]
                     (sort-by #(-> % second :timestamp) > (merge chats browsers))))
+
+(re-frame/reg-sub :chain-sync-state
```
Comment (yenda) [inline src/status_im/ui/screens/home/subs.cljs]: put the name of the subscription on the next line

**Comment by mandrigin** [comment]
@yenda I addressed your feedback ⭐️

File: src/status_im/events.cljs [hunk 3]
```diff
@@ -113,6 +115,13 @@
  (fn [cofx _]
    (init/initialize-keychain cofx)))
 
+;; home screen
+
+(handlers/register-handler-fx
+ :home.ui/sync-info-pressed
+ (fn [cofx name]
```
Comment (yenda) [inline src/status_im/events.cljs]: replace name by _

File: src/status_im/network/core.cljs [hunk 4]
```diff
@@ -128,28 +128,32 @@
...
+                   :failure-event-creator (fn [{:keys [response-body status-code]}]
+                                            (let [reason (if status-code
+                                                           (i18n/label :t/network-invalid-status-code {:code status-code})
+                                                           (str response-body))]
+                                              [::connect-failure {:network-id network-id
+                                                                  :on-failure on-failure
+                                                                  :reason     reason}]))}}
+      (connect-success cofx {:network-id     network-id
+                             :on-success     on-success
+                             :client-version ""}))
```
Comment (yenda) [inline src/status_im/network/core.cljs:156]: no client version ?

File: src/status_im/ui/screens/db.cljs [hunk 5]
```diff
@@ -223,6 +227,9 @@
                  :node/status
                  :node/restart?
                  :node/address
+                 :inbox/fetching?
```
Comment (yenda) [inline src/status_im/ui/screens/db.cljs]: why is inbox/fetching? back ?

File: src/status_im/network/core.cljs (hunk 4 above)
Comment (mandrigin) [reply src/status_im/network/core.cljs:156]: I don't know what does this client version even mean, frankly.
If you run a light node yourself (LES or ULC), you can connect to all sorts of clients, different implementations and different versions, so this field makes no sense.

File: src/status_im/ui/screens/db.cljs (hunk 5 above)
Comment (mandrigin) [reply src/status_im/ui/screens/db.cljs]: probably a merge artifact, will clean it up

**Review by yenda — APPROVED** [review]

**Comment by mandrigin** [comment]
notes for QA — it is important to test this build for regressions in adding custom rpc networks.
the LES indicator might look a bit flaky, and it is fine (because it is a debug UI at the moment)

**Comment by asemiankevich** [comment]
@mandrigin i have been connected to `les.dev.ropsten` LES and switched to `ROPSTEN (LES)` network. The indicator is always `LES: 'latest' #0 not syncing`. Is this something expected? 

image

**Comment by asemiankevich** [comment]
@mandrigin anything on the above?

**Comment by oskarth** [comment]
@asemiankevich FYI @mandrigin is out on vacation for two weeks. Expect delayed response. I believe he mentioned @b00ris would be point person for LES/ULC while he is out, so maybe he wants to pick this PR up?

**Comment by mandrigin** [comment]
@asemiankevich yeah, it looks like the static nodes for LES are different now. I'll fix that, rebase and will ping you.

**Comment by mandrigin** [comment]
@asemiankevich sooo.... ping! :) the latest build is ready to be tested

**Comment by asemiankevich** [comment]
@mandrigin okay i will look into it tomorrow, thanks

**Comment by asemiankevich** [comment]
- syncing indicator is in place
- sync is working

i assume thats it for scope of current PR
image

**Review by rasom — APPROVED** [review]

**Pull request merged**